pkg/openstack/loadbalancer.go

```diff
@@ -1226,6 +1243,12 @@ func (lbaas *LbaasV2) ensureOctaviaPool(lbID string, name string, listener *list
 }
 
 func (lbaas *LbaasV2) buildPoolCreateOpt(listenerProtocol string, service *corev1.Service, svcConf *serviceConfig, name string) v2pools.CreateOpts {
+	var customTag []string
+
+	if svcConf.supportLBTags {
+		customTag = lbaas.getCustomLoadBalancerTags(service)
+	}
```

Comment:
Why don't we check that inside `getCustomLoadBalancerTags()` and if it's `false`, the function will return `nil`? That way you can just directly use `getCustomLoadBalancerTags()` in line 1282.

Reply:
modified already ! but this check will be double checked in a couple of method logic